lib/lumberjack/beats/server.rb

```diff
@@ -66,8 +66,16 @@ def run(&block)
         # unless the server is closing
         next unless connection
 
+        Thread.abort_on_exception = true
```

Comment:
this is a bit scary setting it here.
by default logstash enables abort on exception so this shouldn't be needed.

if it's really necessary I suggest adding an extra check that, if it's false, log a warning that it's being enabled